sycl/ReleaseNotes.md

```diff
@@ -60,7 +60,7 @@ extension. [1d993446] [4f7787c8]
 - Implemented `ext::oneapi::experimental::radix_sorter` from the
 [`sycl_ext_oneapi_group_sort`](doc/extensions/proposed/sycl_ext_oneapi_group_sort.asciidoc)
 extension proposal. [86ba1809]
-- Implemented a new unified interface for the [`sycl_ext_oneapi_matrix`](doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc)
```

Comment:
I don't think we should update the link the point to the latest version of the document. Instead Release Notes should refer to the version we added in 166bbc36.

Reply:
The alternative is to have a broken link in the release notes.  Is that better?

In the past, I have also updated the links in the release notes whenever a file is renamed.  I can stop doing this if you think it's better to leave broken links.

Reply:
> The alternative is to have a broken link in the release notes. Is that better?

No, but what I suggested fixes the repository state, so future change can't break it.
In addition to that I suggest we point the version, which was released. Referring the current version in release notes doesn't make sense to me. NOTE: We even remove the documents from the repository (or rather the content of the documents).

Reply:
Oh, I see.  If we want the links in the release notes to be stable, should we always specify a commit ID when adding them?

Reply:
Right. I think we should use "permanent links" in release notes.
https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files

Reply:
I added the permanent links but note that the commit in the link does not match the commit id in the release note because the commit in the link refers to the documentation while the commit id in the release note refers to the actual development work commit.